jipdate.py

```diff
@@ -191,7 +196,7 @@ def get_jira_issues(jira, exclude_stories, epics_only, all_status, filename,
         f.write("# Header: %s\n" % issue.fields.summary)
         f.write("# Type: %s\n" % issue.fields.issuetype)
         f.write("# Status: %s\n" % issue.fields.status)
-        f.write("No updates since last week.\n\n")
+        f.write("# No updates since last week.\n\n")
```

Comment:
I've pushed the patch(es) enabling a yaml config file (config.yml). The script will create a default file for you that you then can modify. There you can easily add/remove whatever extra comments you would like to see. Btw, you will need to install the YAML Python package also (`pyyaml`). Try it out and see if that sort out the needs from your side.